assets/locale/en.json

```
@@ -99,5 +99,6 @@
   "lang_error_share_native": "Unable to share item.",
   "lang_info_url_copied": "URL copied to clipboard.",
   "lang_close_entry": "Close",
-  "lang_article_actions": "Actions"
+  "lang_article_actions": "Actions",
+  "lang_article_minutes": "min"
```

It would be nice to use so that translators have more control.

```suggestion
  "lang_article_minutes": "{0} min"
```